src/main/java/com/amazonaws/kinesisvideo/util/StreamInfoConstants.java

```diff
@@ -0,0 +1,46 @@
+package com.amazonaws.kinesisvideo.util;
+
+import static com.amazonaws.kinesisvideo.producer.Time.HUNDREDS_OF_NANOS_IN_AN_HOUR;
+import static com.amazonaws.kinesisvideo.producer.Time.HUNDREDS_OF_NANOS_IN_A_SECOND;
+
+public final class StreamInfoConstants {
+    public static final boolean NOT_ADAPTIVE = false;
+    public static final boolean KEYFRAME_FRAGMENTATION = true;
+    public static final boolean SDK_GENERATES_TIMECODES = false;
+    public static final boolean RELATIVE_FRAGMENT_TIMECODES = false;
+    public static final String NO_KMS_KEY_ID = null;
+    public static final int VERSION_ZERO = 0;
+    public static final long MAX_LATENCY_ZERO = 0L;
+    public static final long NO_RETENTION = 0L;
+    public static final long RETENTION_ONE_HOUR = 1L * HUNDREDS_OF_NANOS_IN_AN_HOUR;
+    public static final boolean REQUEST_FRAGMENT_ACKS = true;
+    public static final boolean RECOVER_ON_FAILURE = true;
+    public static final long DEFAULT_GOP_DURATION = 2000L * HUNDREDS_OF_NANOS_IN_A_SECOND;
+    public static final int DEFAULT_BITRATE = 2000000;
+    public static final int DEFAULT_TIMESCALE = 10000;
+    public static final int FRAMERATE_30 = 30;
+    public static final int FRAME_RATE_25 = 25;
```

Comment:
seems different format for FrameRate_30 and 25 (may be enum)?

Reply:
I've fixed the naming to be consistent. I definitely think these should use more enums/value classes, but will leave that for a future PR for now.